spring-petclinic-microservices/README.md

```diff
@@ -26,13 +26,12 @@ or sign up for a
 
 In addition, you will need the following:
 
-| [Azure CLI version 2.0.67 or higher](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
-| [Java 8](https://www.azul.com/downloads/azure-only/zulu/?version=java-8-lts&architecture=x86-64-bit&package=jdk)
-| [Maven](https://maven.apache.org/download.cgi)
-| [MySQL CLI](https://dev.mysql.com/downloads/shell/)
-| [Git](https://git-scm.com/)
-| [Jq](https://stedolan.github.io/jq/)
-|
+- [Azure CLI version 2.0.67 or higher](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+- [Java 8](https://www.azul.com/downloads/azure-only/zulu/?version=java-8-lts&architecture=x86-64-bit&package=jdk)
+- [Maven](https://maven.apache.org/download.cgi)
+- [MySQL CLI](https://dev.mysql.com/downloads/shell/)
+- [Git](https://git-scm.com/)
+- [Jq](https://stedolan.github.io/jq/)
```

Comment:
What does the `Create a new folder and clone the sample app repository to your Azure Cloud account` mean?

Reply:
And do we need this `mkdir source-code`?

Reply:
updated